userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1788,16 +1788,23 @@ static int32_t init(scap_t* handle, scap_open_args *oargs)
 	//
 	// Find out how many devices we have to open, which equals to the number of CPUs
 	//
-	ssize_t num_cpus = sysconf(_SC_NPROCESSORS_ONLN);
+	ssize_t num_cpus = sysconf(_SC_NPROCESSORS_CONF);
```

Comment:
Like in kmod, and like it was before #374 : load all cpus, but only alloc space for online CPUs.